conn.go

```diff
@@ -200,7 +204,7 @@ func (c *conn) handlePgpass(o values) {
 			}
 		}
 		return append(fs, string(f))
-	}	
```

Comment:
if you're upstreaming this (or trying to) it probably makes sense to factor out the drive-by changes in a separate commit.